WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,22 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComPath
+{
```

Comment:
@aerych updated again, so that we support scenarios like this:

```
http(s):*.wordpress.com OR http(s)://wordpress.com
```

Thanks Eric!